### ISSUE TITLE
[$30][MSFT-105] Challenges page: clear filter' and 'save filter' do not receive tab focus.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2809]
**Steps to Reproduce**
1. Go to the Challenges page: https://www.topcoder.com/challenges. 2. Activate the filter. 3. Press the tab key to navigate through the filter controls. 4. Notice that the tab key skips the 'save filter' and 'clear filter' controls.

**Expected Result**
Since these are interactive elements, the 'save filter' and 'clear filter' controls should receive tab focus.

**Actual Result**
The 'save filter' and 'clear filter' controls do not receive tab focus.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.1.1 Keyboard
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2011

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096046 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by nahidshahin** [comment]
@veshu @ThomasKranitsas can't regenerate the problem. The tab is working. After finishing `End Date` `Clear filter` receives the focus. Check the screenshot:

![Selection_068](https://user-images.githubusercontent.com/1224995/61466497-d29c6280-a99b-11e9-9bae-fe4baea04719.jpg)

**Comment by veshu** [comment]
@nahidshahin Please try to navigate with tab without entering any value in filter input items. On disabled state it is not reachable.

**Comment by Oanh-and-only-Oanh** [comment]
@nahidshahin, are you still working on this so I can assign it to you?

**Comment by PrakashDurlabhji** [comment]
@Oanh-and-only-Oanh @veshu can i get this? is it open or nahid will update on it? i wanted to try this one

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096046 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu PR https://github.com/topcoder-platform/topcoder-react-utils/pull/36

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2809 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097277 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097277 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu @sexta13 new PR https://github.com/topcoder-platform/topcoder-react-utils/pull/46

**Comment by crazyk07** [comment]
Closing ticket due to shift in priority, will not be able to complete but will still pay for effort rendered.

Note: Ticket was dependent on react Utils

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097277<br/><br/>```This is an automated message for crazyk via Topcoder X```
